### PR TITLE
CLOUDP-200398: unhide deployments commands

### PR DESCRIPTION
**Pull request by tibulca**
## Proposed changes
Unhide `atlas deployments` commands
Review & LGTM should happen per file. Once the review is completed and all changes implemented please add LGTM comment at the file level (not PR level).

_Jira ticket:_ CLOUDP-200398

### CODE REVIEW

File: docs/atlascli/command/atlas-deployments-connect.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-connect.txt:1]: once the review is completed and all changes implemented please add LGTM at the file level (not PR level)

File: docs/atlascli/command/atlas-deployments-list.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-list.txt:1]: pending for review

File: docs/atlascli/command/atlas-deployments-search-indexes-create.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-search-indexes-create.txt:1]: pending for review

File: docs/atlascli/command/atlas-deployments-search-indexes-describe.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-search-indexes-describe.txt:1]: pending for review

File: docs/atlascli/command/atlas-deployments-search-indexes-list.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-search-indexes-list.txt:1]: pending for review

File: docs/atlascli/command/atlas-deployments-search-indexes.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-search-indexes.txt:1]: pending for review

File: docs/atlascli/command/atlas-deployments-search.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-search.txt:1]: pending for review

File: docs/atlascli/command/atlas-deployments-setup.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-setup.txt:1]: pending for review

File: docs/atlascli/command/atlas-deployments.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments.txt:1]: pending for review

File: docs/atlascli/command/atlas.txt
Comment (tibulca) [inline docs/atlascli/command/atlas.txt:1]: pending for review

File: docs/atlascli/command/atlas-deployments-delete.txt [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+     - Required
+     - Description
+   * - --force
+     - 
+     - false
+     - Flag that indicates whether to skip the confirmation prompt before proceeding with the requested action.
+   * - -h, --help
+     - 
+     - false
+     - help for delete
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-delete.txt:61]:
```suggestion
     - Help for delete.
```

File: docs/atlascli/command/atlas-deployments-connect.txt [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+     - Required
+     - Description
+   * - --connectWith
+     - string
+     - false
+     - Method of connection. Valid values are mongosh, compass and skip.
+   * - -h, --help
+     - 
+     - false
+     - help for connect
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-connect.txt:61]:
```suggestion
     - Help for connect.
```

File: docs/atlascli/command/atlas-deployments-list.txt [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for list
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-list.txt:41]:
```suggestion
     - Help for list.
```

File: docs/atlascli/command/atlas-deployments-search-indexes-describe.txt [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+     - Required
+     - Description
+   * - --deploymentName
+     - string
+     - false
+     - Name of the deployment.
+   * - -h, --help
+     - 
+     - false
+     - help for describe
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-search-indexes-describe.txt:61]:
```suggestion
     - Help for describe.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+     - false
+     - Method of connection. Valid values are mongosh, compass and skip.
+   * - --force
+     - 
+     - false
+     - Flag that indicates whether to skip the confirmation prompt before proceeding with the requested action.
+   * - -h, --help
+     - 
+     - false
+     - help for setup
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:65]:
```suggestion
     - Help for setup.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - deploymentName
+     - string
+     - false
+     - Name of the deployment you want to setup.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:41]:
```suggestion
     - Name of the deployment that you want to setup.
```

**Review by corryroot — COMMENTED** [review]
Thanks @tibulca! I commented with copy nits.

File: docs/atlascli/command/atlas.txt [hunk 7]
```diff
@@ -64,6 +64,7 @@ Related Commands
 * :ref:`atlas-dataFederation` - Data federation.
 * :ref:`atlas-dataLakePipelines` - Data Lake pipelines.
 * :ref:`atlas-dbusers` - Manage database users for your project.
+* :ref:`atlas-deployments` - Manage Atlas and local deployments.
```
Comment (corryroot) [inline docs/atlascli/command/atlas.txt:67]:
```suggestion
* :ref:`atlas-deployments` - Manage cloud and local deployments.
```

File: docs/atlascli/command/atlas-deployments-connect.txt (hunk 2 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-connect.txt:61]: the --help flag (and the documentation) is not controlled by us, but by the [cobra lib](https://github.com/spf13/cobra/blob/0c72800b8dba637092b57a955ecee75949e79a73/command.go#L1164).
For all existing cli commands the --help is documented using the same format "help for <cmd>".
Example: https://www.mongodb.com/docs/atlas/cli/stable/command/atlas-accessLists-list/

File: docs/atlascli/command/atlas-deployments-connect.txt [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Connects to a deployment.
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-connect.txt]: done, thanks

File: docs/atlascli/command/atlas-deployments-connect.txt [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - deploymentName
+     - string
+     - false
+     - Name of the deployment you want to connect.
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-connect.txt]: done, thanks

File: docs/atlascli/command/atlas-deployments-connect.txt [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --connectWith
+     - string
+     - false
+     - Method of connection. Valid values are mongosh, compass and skip.
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-connect.txt]: done, thanks

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 6 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done, thanks

File: docs/atlascli/command/atlas.txt (hunk 7 above)
Comment (tibulca) [reply docs/atlascli/command/atlas.txt]: done, thanks

File: docs/atlascli/command/atlas-deployments-search-indexes-delete.txt
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-search-indexes-delete.txt:1]: pending for review - Sept 19th

File: docs/atlascli/command/atlas-deployments-connect.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-connect.txt:1]: LGTM

File: docs/atlascli/command/atlas-deployments-delete.txt
[the remark this answers is not in the document]
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-delete.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-list.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-list.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-search-indexes-create.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-search-indexes-create.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-search-indexes-delete.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-search-indexes-delete.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-search-indexes-describe.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-search-indexes-describe.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-search-indexes-list.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-search-indexes-list.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-search-indexes.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-search-indexes.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - deploymentName
+     - string
+     - false
+     - Name of the deployment that you want to setup.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:41]:
```suggestion
     - Name of the deployment that you want to set up.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --accessListIp
+     - strings
+     - false
+     - IP address to be allowed to access the deployment.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:57]:
```suggestion
     - IP address to grant access to the deployment.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --provider
+     - string
+     - false
+     - Name of your cloud service provider. Valid values are AWS, AZURE, or GCP.
+   * - -r, --region
+     - string
+     - false
+     - Physical location of your MongoDB cluster. For a complete list of supported AWS regions, see: https://www.mongodb.com/docs/atlas/reference/amazon-aws/. For a complete list of supported Azure regions, see: https://www.mongodb.com/docs/atlas/reference/microsoft-azure/#microsoft-azure. For a complete list of supported GCP regions, see: https://www.mongodb.com/docs/atlas/reference/google-gcp/..
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:105]: We should use dochub URLs.

```suggestion
     - Physical location of your MongoDB cluster. For a complete list of supported AWS regions, see: https://dochub.mongodb.org/core/aws-atlas. For a complete list of supported Azure regions, see: https://dochub.mongodb.org/core/azure-atlas. For a complete list of supported GCP regions, see: https://dochub.mongodb.org/core/gcp-atlas.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+   * - --skipMongosh
+     - 
+     - false
+     - Indicates whether to skip accessing your deployment with MongoDB Shell.
+   * - --skipSampleData
+     - 
+     - false
+     - Indicates whether to skip loading sample data into your Atlas cluster.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:113]:
```suggestion
     - Indicates whether to skip loading sample data into your MongoDB cluster.
```

File: docs/atlascli/command/atlas-deployments.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments.txt:1]: LGTM!

File: docs/atlascli/command/atlas.txt
Comment (corryroot) [reply docs/atlascli/command/atlas.txt:1]: LGTM!

**Review by corryroot — COMMENTED** [review]
Most of these look good! I have some nits for `atlas deployments setup`.

File: docs/atlascli/command/atlas-deployments-delete.txt (hunk 1 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-delete.txt:61]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments-list.txt (hunk 3 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-list.txt:41]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments-search-indexes-create.txt [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+   * - -f, --file
+     - string
+     - false
+     - Name of the JSON index configuration file to use. To learn about the Atlas Search index configuration file, see https://www.mongodb.com/docs/atlas/cli/stable/reference/json/search-index-config-file/. To learn about the Atlas Search index syntax and options that you can define in your configuration file, see https://www.mongodb.com/docs/atlas/atlas-search/index-definitions/.
+
+       Mutually exclusive with --db, --collection.
+   * - -h, --help
+     - 
+     - false
+     - help for create
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search-indexes-create.txt:79]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments-search-indexes-describe.txt (hunk 4 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search-indexes-describe.txt:61]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 5 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt:81]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments.txt [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for deployments
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments.txt:31]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments-search.txt [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for search
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search.txt:31]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments-search-indexes.txt [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for indexes
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search-indexes.txt:31]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments-search-indexes-list.txt [hunk 19]
```diff
@@ -0,0 +1,85 @@
...
+     - false
+     - Name of the database.
+   * - --deploymentName
+     - string
+     - false
+     - Name of the deployment.
+   * - -h, --help
+     - 
+     - false
+     - help for list
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search-indexes-list.txt:53]: same as https://github.com/mongodb/mongodb-atlas-cli/pull/2263/files#r1329411943

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 12 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done, thanks

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 11 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done, thanks

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 13 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done, thank you

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 14 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done, thank you

File: docs/atlascli/command/atlas-deployments-search-indexes-delete.txt [hunk 20]
```diff
@@ -0,0 +1,95 @@
+.. _atlas-deployments-search-indexes-delete:
```
Comment (tibulca) [inline docs/atlascli/command/atlas-deployments-search-indexes-delete.txt:1]: pending for review: Sept 19th

File: docs/atlascli/command/atlas-deployments-search-indexes-delete.txt [hunk 21]
```diff
@@ -0,0 +1,95 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Delete the specified search index from the specified cluster.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-search-indexes-delete.txt:15]: In the docs, we've replaced cluster with deployments throughout.

```suggestion
Delete the specified search index from the specified deployment.
```

File: docs/atlascli/command/atlas-deployments-search.txt [hunk 22]
```diff
@@ -0,0 +1,59 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Manage search for Atlas and local deployments.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-search.txt:15]:
```suggestion
Manage search for cloud and local deployments.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 23]
```diff
@@ -0,0 +1,146 @@
...
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --provider
+     - string
+     - false
+     - Name of your cloud service provider. Valid values are AWS, AZURE, or GCP.
+   * - -r, --region
+     - string
+     - false
+     - Physical location of your MongoDB cluster. For a complete list of supported AWS regions, see: https://dochub.mongodb.org/core/aws-atlas. For a complete list of supported Azure regions, see: https://dochub.mongodb.org/core/azure-atlas. For a complete list of supported GCP regions, see: https://dochub.mongodb.org/core/gcp-atlas..
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:105]:
```suggestion
     - Physical location of your MongoDB deployment. For a complete list of supported AWS regions, see: https://dochub.mongodb.org/core/aws-atlas. For a complete list of supported Azure regions, see: https://dochub.mongodb.org/core/azure-atlas. For a complete list of supported GCP regions, see: https://dochub.mongodb.org/core/gcp-atlas..
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 24]
```diff
@@ -0,0 +1,146 @@
...
+   * - --skipMongosh
+     - 
+     - false
+     - Indicates whether to skip accessing your deployment with MongoDB Shell.
+   * - --skipSampleData
+     - 
+     - false
+     - Indicates whether to skip loading sample data into your MongoDB cluster.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:113]:
```suggestion
     - Indicates whether to skip loading sample data into your MongoDB deployment.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 25]
```diff
@@ -0,0 +1,146 @@
...
+   * - --tag
+     - stringToString
+     - false
+     - List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster. This value defaults to [].
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:117]:
```suggestion
     - List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the deployment. This value defaults to [].
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 26]
```diff
@@ -0,0 +1,146 @@
...
+   * - --tier
+     - string
+     - false
+     - Tier for each data-bearing server in the cluster. To learn more about cluster tiers, see https://www.mongodb.com/docs/atlas/manage-clusters/#select-cluster-tier. This value defaults to "M0".
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:121]:
```suggestion
     - Tier for each data-bearing server in the deployment. To learn more about cluster tiers, see https://dochub.mongodb.org/core/cluster-tier-atlas. This value defaults to "M0".
```

File: docs/atlascli/command/atlas-deployments-search-indexes-delete.txt [hunk 27]
```diff
@@ -0,0 +1,95 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --deploymentName
+     - string
+     - false
+     - Name of the cluster.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-search-indexes-delete.txt:57]:
```suggestion
     - Name of the deployment.
```

File: docs/atlascli/command/atlas-deployments-search-indexes.txt [hunk 28]
```diff
@@ -0,0 +1,65 @@
...
+   * - -P, --profile
+     - string
+     - false
+     - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
+
+Related Commands
+----------------
+
+* :ref:`atlas-deployments-search-indexes-create` - Create a search index for the specified deployment.
+* :ref:`atlas-deployments-search-indexes-delete` - Delete the specified search index from the specified cluster.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-search-indexes.txt:53]:
```suggestion
* :ref:`atlas-deployments-search-indexes-delete` - Delete the specified search index from the specified deployment.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 29]
```diff
@@ -0,0 +1,146 @@
...
+   * - --currentIp
+     - 
+     - false
+     - Flag that adds the IP address from the host that is currently executing the command to the access list.
+
+       Mutually exclusive with --accessListIp.
+   * - --enableTerminationProtection
+     - 
+     - false
+     - Enables termination protection for your cluster. You can't delete a cluster with termination protection enabled.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:73]:
```suggestion
     - Enables termination protection for your deployment. You can't delete a deployment with termination protection enabled.
```

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 30]
```diff
@@ -0,0 +1,146 @@
...
+   * - --force
+     - 
+     - false
+     - If specified, skips asking for input and creates a cluster with default settings for any options you don't specify.
```
Comment (corryroot) [inline docs/atlascli/command/atlas-deployments-setup.txt:77]:
```suggestion
     - If specified, skips asking for input and creates a deployment with default settings for any options you don't specify.
```

**Review by corryroot — COMMENTED** [review]
These look really good! For the docs, we ended up changing all references to clusters to deployments and I wonder whether we should do the same here.

File: docs/atlascli/command/atlas-deployments-setup.txt [hunk 31]
```diff
@@ -0,0 +1,146 @@
...
+   * - -h, --help
+     - 
+     - false
+     - help for setup
+   * - --mdbVersion
+     - string
+     - false
+     - Major MongoDB version of the cluster.
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done, thank you

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 30 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done, thank you

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 29 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done, thank you

File: docs/atlascli/command/atlas-deployments-search-indexes.txt (hunk 28 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search-indexes.txt]: done, thank you

File: docs/atlascli/command/atlas-deployments-search-indexes-delete.txt (hunk 27 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search-indexes-delete.txt]: done, thank you

File: docs/atlascli/command/atlas-deployments-search-indexes-delete.txt (hunk 21 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search-indexes-delete.txt]: done

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 23 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 25 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done

File: docs/atlascli/command/atlas-deployments-search.txt (hunk 22 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search.txt]: done

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 24 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done

File: docs/atlascli/command/atlas-deployments-setup.txt (hunk 26 above)
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-setup.txt]: done

File: docs/atlascli/command/atlas-deployments-search-indexes-delete.txt (hunk 20 above)
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-search-indexes-delete.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-setup.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-setup.txt:1]: LGTM!

File: docs/atlascli/command/atlas-deployments-search.txt
Comment (corryroot) [reply docs/atlascli/command/atlas-deployments-search.txt:1]: LGTM. One nit.

File: docs/atlascli/command/atlas-deployments-search.txt [hunk 32]
```diff
@@ -0,0 +1,59 @@
...
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
+
+Related Commands
+----------------
+
+* :ref:`atlas-deployments-search-indexes` - Manage Atlas and local search indexes.
```
[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search.txt:52]: resolving in a follow-up PR, thank you

[the remark this answers is not in the document]
Comment (tibulca) [reply docs/atlascli/command/atlas-deployments-search.txt:52]: https://github.com/mongodb/mongodb-atlas-cli/pull/2304/files

**Pull request merged**